server/motr_common.cc

```diff
@@ -168,6 +168,15 @@ void teardown_motr_op(struct m0_op *op) { teardown_motr_cancel_wait_op(op, 0); }
 void teardown_motr_cancel_wait_op(struct m0_op *op, int sec) {
   if (!op) return;
 
+  // Free fake m0_op for read in case of faked create and open
+  // NOTE: during operation init m0_op should be allocated accordingly
+  // see s3_motr_wrapper.cc::ConcreteMotrAPI::motr_obj_op
+  if (S3Option::get_instance()->is_fake_motr_obj_op_read(
+          (m0_obj_opcode)op->op_code)) {
+    free(op);
```

Comment:
do we need to assign nullptr to 'op' after 'free' call ?

Reply:
we can, but it is not necessary since this func is in the middle of the parent object cleanup logic